Typo "Tensorflow"→"TensorFlow"

hyoshioka0128 opened the pull request:
https://docs.microsoft.com/en-us/windows/ai/directml/gpu-tensorflow-wsl
https://github.com/MicrosoftDocs/windows-ai-docs/blob/docs/docs/directml/gpu-tensorflow-wsl.md
#PingMSFTDocs

opbld31 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/winmlrunner?branch=pr-en-us-50) |
[docs/winmlrunner.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/winmlrunner.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/winmlrunner?branch=pr-en-us-50) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C23%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201810232255070381-50%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kdHa4gFE8SQT6RwCnKKG5KDv%2FUTB%2BYlaoYAEASBdN54%3D&st=2018-10-23T22%3A51%3A23Z&se=2018-11-23T22%3A56%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld32 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/winmlrunner?branch=pr-en-us-50) |
[docs/winmlrunner.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/winmlrunner.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/winmlrunner?branch=pr-en-us-50) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C25%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201810252237282788-50%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=GWiI7xLEngOdVZSzKvy8ZoIOEXni7vZXVyH58jYZweQ%3D&st=2018-10-25T22%3A33%3A43Z&se=2018-11-25T22%3A38%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld34 commented:
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/TOC.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/windows/ai/tools-and-samples?branch=pr-en-us-50) | [Details](#user-content-docs%2FTOC.md)
[docs/tools-and-samples.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/tools-and-samples.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/tools-and-samples?branch=pr-en-us-50) |

<a name="docs%2FTOC.md"></a>
### [docs/TOC.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/TOC.md)
 - **Line 0**: **[Warning]** Invalid file link:(~/winmlrunner.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C26%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201810260333003006-50%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=q8LPwDbniXShxlvrjJe%2BV2NKNhX320cw5QBuFL3jvss%3D&st=2018-10-26T03%3A29%3A14Z&se=2018-11-26T03%3A34%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld31 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/tools-and-samples?branch=pr-en-us-50) |
[docs/tools-and-samples.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/tools-and-samples.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/tools-and-samples?branch=pr-en-us-50) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C26%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201810262204281603-50%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ykXFS4QVzQpNqP0PbWXyRCc8LEQZ38gkKcPruyk%2FdbE%3D&st=2018-10-26T22%3A00%3A42Z&se=2018-11-26T22%3A05%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld34 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/tools-and-samples?branch=pr-en-us-50) |
[docs/api-reference.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/api-reference.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/api-reference?branch=pr-en-us-50) |
[docs/tools-and-samples.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/tools-and-samples.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/tools-and-samples?branch=pr-en-us-50) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C26%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201810262253596018-50%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=GI5W26cKBQaT94IzlLboMxZ4oVkesq%2Fee9AxwBkhe8E%3D&st=2018-10-26T22%3A50%3A13Z&se=2018-11-26T22%3A55%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld32 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/tools-and-samples?branch=pr-en-us-50) |
[docs/api-reference.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/api-reference.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/api-reference?branch=pr-en-us-50) |
[docs/tools-and-samples.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/elcowle-master-winmlrunner/docs/tools-and-samples.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/tools-and-samples?branch=pr-en-us-50) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C29%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201810291801083337-50%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OZ0F%2F8jfSGiEAoOnKFWxb680be47nq8eXhhv%2FWfvvQ4%3D&st=2018-10-29T17%3A57%3A22Z&se=2018-11-29T18%3A02%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

hyoshioka0128 committed:
Typo "Tensorflow"→"TensorFlow"

https://docs.microsoft.com/en-us/windows/ai/directml/gpu-tensorflow-wsl
https://github.com/MicrosoftDocs/windows-ai-docs/blob/docs/docs/directml/gpu-tensorflow-wsl.md
#PingMSFTDocs

stevewhims reviewed (approved):
Thank you!
